packages/modules/manage-versions/src/components/ManageVersions/VersionsTab/VersionsTab.tsx

```diff
@@ -29,17 +30,16 @@ const VersionsTab = (props: VersionsTabProps) => {
     onVersionUpdated,
     loadMoreVersions,
     onViewClick,
+    isLoadingColumn,
```

Comment:
Please name it more specifically.. WHich column is loading?

Reply:
removed this prop